source/chapters/appendix/settings_directory.rst

```diff
@@ -8,28 +8,26 @@ The Mixxx Settings Directory
 .. sectionauthor::
    ronso0 <ronso0@mixxx.org>
 
-The Mixxx settings directory contains all user data and settings of your Mixxx installation.
+The Mixxx settings directory contains all user data and settings of your Mixxx installation. If you want to make a backup of your library and all preferences copy the entire directory to a safe place.
 
 Location
 --------
 
-**Linux**
-
-    ``~/.mixxx/``
-
-    **Note:** make sure you have "Show hidden files" enabled in your file manager
-    in order to show all dot-files and directories.
+.. hint:: On Windows, the :file:`AppData` folder is hidden, as well as all files and
+          directories beginning with a dot '.' on Linux. So if you have not already,
+          you will need to set your file manager to show hidden files and folders.
 
 **Windows**
-
-    | Vista and up: :file:`%LOCALAPPDATA%\\Mixxx`
-    | XP and below: :file:`%USERPROFILE%\\Local Settings\\Application Data\\Mixxx\\`
-    | Type either of those into the location bar of a Computer or Folder window.
+ | Vista and up: ``%LOCALAPPDATA%\Mixxx``
+ | XP and below: ``%USERPROFILE%\Local Settings\Application Data\Mixxx``
```

Comment:
We have dropped Win XP Support. I am not sure if it worth to keep it, but if we decide too keep it for users coming from XP upgrading to a recent version. We should move it ino a separate sentence describing that. 
I would remove the version dependency from the main sentence to make it less confusing.

Reply:
Okay, makes sense. I can't empathize with Windows users anymore, it's just been too long since I had to use it ;)

Reply:
Looks like this now
![image](https://user-images.githubusercontent.com/5934199/151838814-f8d014e5-114a-4099-8f21-b5dbc2187e4d.png)